packages/teams-js/src/private/store.ts

```diff
@@ -110,6 +110,15 @@ export interface OpenSpecificStoreParams extends StoreSizeInfo {
  */
 export type OpenStoreParams = OpenFullStoreAndICSParams | OpenAppDetailParams | OpenSpecificStoreParams;
 
+/**
+ * @beta
+ * @hidden
+ * error message when getting illegal store dialog size
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export const errorInvalidDialogSize = 'Invalid store dialog size';
```

Comment:
```suggestion
const errorInvalidDialogSize = 'Invalid store dialog size';
```

No need to export this. I'd actually like this to be moved into the function it's used in as well. Since we aren't using this string anywhere else there's no need to put it in the global namespace

Reply:
This was resolved but no change was made and no explanation was given as to why no change was made.

Reply:
This error message is be veried in unit test, so I make it export, like other error scenario.